demos/markdown/tutorial/tutorial.didact.md

```diff
@@ -0,0 +1,129 @@
+![Writing Your First Didact Tutorial](images/header.png){.imageCenter}
+
+# Welcome!
+
+This Didact tutorial uses Didact itself to help you create your first Didact tutorial!
+
+## What is Didact?
+
+Didact is a framework that fills a need for a more active kind of tutorial or click-through document in VS Code.
+
+Do you find yourself writing documents for yourself or others filled with steps to accomplish various tasks? Do you write them in Markdown or AsciiDoc for easier formatting to share them with others? Do you find yourself wishing they could do more than be just static repositories full of chunks of text to copy and paste in various ways?
+
+Didact gives you a way of writing those same files but gives them a way to leverage commands in VS Code to do a lot more than copy and paste!
+
+## Getting Started with Didact
+
+1. Install Microsoft Visual Studio Code (you already have this!) [Visual Studio Code website](https://code.visualstudio.com/)
+2. Add the [vscode-didact extension](vscode:extension/redhat.vscode-didact) (you already have this too if you're seeing the tutorial in the `Didact Tutorials` view!)
+3. Create a new AsciiDoc or Markdown file and add “didact” to the file extension (`myfile.didact.adoc`, `myotherfile.didact.md`) [(Click here to create your first Didact markdown file named `myfirst.didact.md`.)](didact://?commandId=vscode.didact.scaffoldProject&extFilePath=redhat.vscode-didact/demos/markdown/tutorial/didactmdfile.json)
+4. Start writing!
```

Comment:
the file is created but not opened when using the link.
Maybe worth either to open it or to mention to open it

Reply:
Right now there isn't a way to do a combined create/open. Would be a good feature. (created https://issues.redhat.com/browse/FUSETOOLS2-742) 

Will see if I can add some text around ensuring that the user opens the link or we can add another didact link to handle it.